hdl/XilinxOutOfContextConstraints.tmpl

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu -o pipefail
+
+cat > {{ .Out }} <<EOF
```

Comment:
Please rewrite to use the `Data` member in `AddBuildStep`. Then you don't need to create a script that generates data.

Reply:
Thanks.